src/server/animals/controllers/status-poller.js

```diff
@@ -44,7 +42,7 @@ const statusPollerController = {
 
       // TODO: save the file info to the session
       await saveToAnimal(request, h, {
-        fileUrl: fileStatus.uploadId + '/' + fileStatus.fileId
+        fileUrl: encodeURI(status.files[0].s3Key)
```

Comment:
This is the wrong way to do it. The reason its 404ing is a slash is missing in the path in the router. The encodeURI just hides that path. So IMO its not the right way to fix this. What you need to do is see how many slashes/parts are in the `fileUrl` and match that in the `path` in the `src/server/files/index.js` router. Then this is easier to understand